File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v4/repository/PostRepository.java
package com.example.dtomappingbenchmark.v4.repository;

import com.example.dtomappingbenchmark.v4.entity.Post;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository("postRepositoryV4")
public interface PostRepository extends JpaRepository<Post, Long> {
}

File: object-mapping-benchmark-v4/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv4/post/repository/PostRepository.kt
package yu.selab.cj.objectmappingbenchmarkv4.post.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import yu.selab.cj.objectmappingbenchmarkv4.post.entity.Post


@Repository
interface PostRepository : JpaRepository<Post, Long> {
}